test/serialize.cpp

```diff
@@ -340,3 +340,53 @@ TEST(serialize, non_zero_node_output)
     EXPECT_EQ(topk_out.get_index(), 1);
     EXPECT_EQ(topk_out.get_node()->description(), "TopK");
 }
+
+TEST(serialize, tensor_iterator)
+{
+    // Common to all cells
+    auto X = make_shared<op::Parameter>(element::f32, Shape{32, 40, 10});
+    auto Hinit = make_shared<op::Parameter>(element::f32, Shape{20});
+    auto WH = make_shared<op::Parameter>(element::f32, Shape{20, 20});
+    auto WX = make_shared<op::Parameter>(element::f32, Shape{10, 20});
+    auto bH = make_shared<op::Parameter>(element::f32, Shape{20});
+    auto WY = make_shared<op::Parameter>(element::f32, Shape{20, 5});
+    auto bY = make_shared<op::Parameter>(element::f32, Shape{5});
+
+    // Set up the cell body, a function from (Hi, Xi) -> (Ho, Yo)
+    // Cell parameters
+    auto Hi = make_shared<op::Parameter>(element::f32, Shape{32, 20});
+    auto Xi = make_shared<op::Parameter>(element::f32, Shape{32, 1, 10});
+
+    // Body
+    auto Ho = make_shared<op::Relu>(
+        make_shared<op::Dot>(make_shared<op::Reshape>(Xi, AxisVector{0, 1, 2}, Shape{32, 10}), WX) +
+        make_shared<op::Dot>(Hi, WH) + make_shared<op::Broadcast>(bH, Shape{32, 20}, AxisSet{0}));
+    auto Yo = make_shared<op::Relu>(make_shared<op::Dot>(Ho, WY) +
+                                    make_shared<op::Broadcast>(bY, Shape{32, 5}, AxisSet{0}));
+
+    // TensorIterator
+    // axis=1, start=0, stride=1, part_size=1, end=40
+    auto Xseq = make_shared<op::SliceInput>(X, 1, 0, 1, 1, 40);
+    auto Hseq = make_shared<op::SequenceRepeat>(Ho);
+    auto Yseq = make_shared<op::SequenceRepeat>(Yo);
+    auto tensor_iterator = make_shared<op::TensorIterator>(
+        // Body inputs
+        OutputVector{
+            Xseq,                                            // Xi values
+            make_shared<op::SequencePushFront>(Hinit, Hseq), // Hi values
+        },
+        ParameterVector{Xi, Hi}, // Body parameters
```

Comment:
Is this ParameterVector contains initial values?

Reply:
These are the body parameters.